hosting/src/hooks/useFirebaseUi.tsx

```diff
@@ -31,6 +31,10 @@ export function useFirebaseUi() {
       tosUrl: "https://github.com/oddbit/tanam/blob/main/docs/tos.md",
       privacyPolicyUrl: "https://github.com/oddbit/tanam/blob/main/docs/privacy-policy.md",
       signInOptions: [
+        {
+          provider: EmailAuthProvider.PROVIDER_ID,
+          fullLabel: isSignUp ? "Sign up with email" : "Sign in with email",
+        },
```

Comment:
Adding email sign in is not part of the scope

Reply:
This is just a change in indentation when doing `yarn codecheck`

Reply:
You added it
https://github.com/oddbit/tanam/commit/fb88307cdd8e4ec05308290e716dfb55f32b40bd#diff-f36bd848e1db75f098c291aa1e4c34458f2103c6ac5ea1cf2f8bd74ed7decc5f

Reply:
Sorry, it seems there was a mistake from me because I forgot to delete the testing code. I thought it was just because of the indentation problem and I forgot to add it. I also forgot if we don't have a login with email button.
I'll delete it in the next update